cmd/minikube/cmd/start_flags.go

```diff
@@ -153,6 +154,7 @@ func initMinikubeFlags() {
 // initKubernetesFlags inits the commandline flags for Kubernetes related options
 func initKubernetesFlags() {
 	startCmd.Flags().String(kubernetesVersion, "", fmt.Sprintf("The Kubernetes version that the minikube VM will use (ex: v1.2.3, 'stable' for %s, 'latest' for %s). Defaults to 'stable'.", constants.DefaultKubernetesVersion, constants.NewestKubernetesVersion))
+	startCmd.Flags().String(startNamespace, "default", "The namespace used in the Kubernetes cluster")
```

Comment:
a better description would be, named space to activate after start.

Reply:
i've changed the description accordingly